conda_forge_tick/migrators/core.py

```diff
@@ -721,7 +721,18 @@ def commit_message(self, feedstock_ctx: FeedstockContext) -> str:
 
     def pr_title(self, feedstock_ctx: FeedstockContext) -> str:
         assert isinstance(feedstock_ctx.attrs["new_version"], str)
-        return feedstock_ctx.package_name + " v" + feedstock_ctx.attrs["new_version"]
+        # TODO remove this later when we want to automerge everything
+        if feedstock_ctx.feedstock_name == 'cf-autotick-bot-test-package':
```

Comment:
```suggestion
         # TODO: turn False to True when we default to automerge
        if feedstock_ctx.attrs.get('conda-forge.yml', {}).get('bot', {}).get('automerge', False):
```

Reply:
This way we don't actually need to do anything as automerge is enabled

Reply:
Right. See my above comment about keeping this logic only in the automerge bot.

Reply:
Ok

Reply:
So without this we can still turn it on globally. We simply switch the default here: https://github.com/regro/cf-autotick-bot-action/blob/master/conda_forge_tick_action/automerge.py#L29 and then redeploy the bot.

Reply:
We could use the ternary logic and bump the switch in both spots.

Reply:
ah ok

Reply:
Sorry for being terse earlier. You are totally right. What I want to avoid is us having to go through each feedstock and rerender it. As long as we can change the default on the backend, we will be fine. This change allows that. Plus having the switch in two spots is probably a good idea.

Reply:
Would it be worthwhile going the other way. Label PRs which can't be automerged?
That way the automerge is set to the feedstock wide default unless a particular PR wants not to be automerged.

Reply:
Hmmmm. I prefer adding the slug for automerge as opposed to adding a slug to not automerge for now. I think this change might make sense later as this gets further into deployment?